tox.ini

```diff
@@ -90,3 +91,9 @@ deps={[common-wheel-cli]deps}
 allowlist_externals={[common-wheel-cli]allowlist_externals}
 commands={[common-wheel-cli]commands}
 skip_install=true
+
+[testenv:py311-wheel-cli]
```

Comment:
This would be a good time to trim all of these `wheel-cli` jobs to one group:

```ini
[testenv:py{37,38,39,310,311}-wheel-cli]
deps={[common-wheel-cli]deps}
allowlist_externals={[common-wheel-cli]allowlist_externals}
commands={[common-wheel-cli]commands}
skip_install=true
```